regclient.yaml

```
@@ -0,0 +1,33 @@
+package:
+  name: regclient
+  version: 0.4.5
+  epoch: 0
+  description:  Docker and OCI Registry Client in Go and tooling using those libraries
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/regclient/regclient/archive/v${{package.version}}/v${{package.version}}.tar.gz
+      expected-sha256: 298c6b8e6afd1557476339e5a5985ecf581b60816149e25fa7d1bcfe396b2a4f
+  - runs: |
+      CGO_ENABLED=0 go build \
+        -trimpath -ldflags "-s -w -extldflags -static -buildid=" \
+        -o "${{targets.destdir}}/usr/bin/regctl" ./cmd/regctl
```

Is there some reason we aren't stripping the binary afterward?

```suggestion
        -o "${{targets.destdir}}/usr/bin/regctl" ./cmd/regctl
  - uses: strip
```